### DISCUSSION TITLE
Samsung Odyssey G7 27'' via DP 'dead' after sleep

### DISCUSSION THREAD

**Discussion by doncarlos1551** [xanderfrangos/twinkle-tray#729] [Monitor Compatibility Issues]
# My System
- twinkle tray v1.15.4 installed from the microsoft store 
- windows 11 pro Version 10.0.22631 Build 22631
- rtx 3070 with GeForce Game Ready Driver 551.52
- triple samsung odyssey G7 27'' w model no LC27G75TQSPXEN (1 HDMI, 2 DP) - currently seemingly no [firmware update](https://www.samsung.com/de/support/model/LC27G75TQSPXEN/#downloads) available

# Bug
When twinkle tray is running and my windows 11 will go to sleep my monitors that are connected via DP will not wake up, they also will not be responding to using the hardware knob/button. When connected via HDMI the problem does not occur so this problem is *display port only*. Only way to bring them back is unplugging and replugging their power cord. When twinkle tray is closed/not running the problem is gone. I tried disabling/enabling all the options in troubleshooting, also tried to activate the power state control, did not change anything. Also tried playing with different settings in the monitors settings with no avail.

According to this [reddit post](https://www.reddit.com/r/Monitors/comments/14silyz/samsung_g7_odyssey_doesnt_wake_up_from_the_sleep/) and further this [reddit comment](https://www.reddit.com/r/Monitors/comments/14silyz/comment/k85nx8l/?utm_source=share&utm_medium=web2x&context=3) it has to do with deep sleep. It seems to be fixed in windows 11, but comes back through twinkle tray somehow. At least thats my supposition.

Using monitorian I am not having the problem described above. I did not try any other software so far.

Edit: Everything else seems to be working really well.

**Comment by pureby** [comment]
Try experimenting with these two settings: 
1. DDC/CI ▸ Power state
2. Hotkeys & shortcuts ▸ Turn off display action

**Reply by doncarlos1551** [reply]
1. I tried activating power state already as stated above, didn't change anything sadly...
2. Won't the selection of "Turn off Displays action" only be used for when I use the specified hotkey and turn off the display via twinkle tray? My Problem is when the sleep is triggered through windows, while twinkle tray is running. Nevertheless I'll give it a try and will give an update later!

Update:
Tried changing "Turn off Displays action" to "Hardware signal (DDC/CI only)" and "Hardware & Software signal" and both did the same for when windows does the display sleep.
When I use the hotkey or the twinkle tray turn off displays icon to send my displays to sleep all the three options (software, hardware and mixed) will work totally fine, problem will only be when Windows does it, also doesn't matter which one of these three is picked.

**Reply by pureby** [reply]
Thanks for the quick feedback. This behavior is strange. Are you using a desktop or a laptop? If you're on a laptop, there might be some manufacturer software or driver conflicting with TwinkleTray. Try closing all manufacturer-related software and drivers (including background services using Task Manager), and see if that fixes the problem.

**Reply by doncarlos1551** [reply]
I am using a desktop pc, if you need my complete build information I can give you that. I have only the mainboard manufacturer(gigabyte) drivers, the amd cpu drivers and for the gpu the nvidia game ready driver via geforce experience installed.

In the reddit links provided in the main post this is a common problem for many with the g7s in multi monitor setup, but some say it was fixed for win 11. This is the case for me as well, but only when twinkle tray is turned on this problem will occure for me. 

Yesterday I had a call with Samsung and they seem to know this issue, they told me many other customers have this problem(unrelated to twinkle tray) still with win 10 and win 11 and they confirmed it has to do with deep sleep of the monitor; Their advice was to disable sleep entirely. If you have any other idea how to fix this or create a workaround I will gladly try it out, as I would love going back to twinkle tray.
